### DISCUSSION TITLE
self-hosted instance: No user registration link ?

### DISCUSSION THREAD

**Discussion by davidsean** [biigle/core#811] [Q&A] [answered]
Hello,
I`ve been trying out the docker compose deployments of Biigle and noticed I could not find the user self-registration URL. Of course as a super-user I can add new users myself, but is there a way for users to register for themselves ? I've look through the config options  (build environmental variables ) to no avail. Any tips appreciated, thanks!

**Comment by mzur** [comment, marked as answer]
Hi, you can find "hidden" config options in the [biigle/core](https://github.com/biigle/core) config directory. The one for the user registration is located in [`biigle.php`](https://github.com/biigle/core/blob/e679cf3778f393fedad9722ef9ad3c919a102f18/config/biigle.php#L28) and called `BIIGLE_USER_REGISTRATION`.

You can enable this by adding `BIIGLE_USER_REGISTRATION=true` to the `build/.env` file of your BIIGLE instance.

**Reply by davidsean** [reply]
Added the variable, re-built, and now the sign-up link works. Thanks Martin.
